bom/deployment/pom.xml

```diff
@@ -654,6 +654,11 @@
                 <artifactId>quarkus-mutiny-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-mutiny-reactive-streams-operators-deployment</artifactId>
```

Comment:
Is there a limit to the length of an artifact name? :)

Reply:
Nope :-) Well, until it fits in the memory of the JVM.

Note that it's internal only. No one except us is going to use this.